packages/documents/lib/resolve.js

```diff
@@ -177,11 +177,19 @@ const metaFieldResolver = (meta, allDocuments = [], errors) => {
     }
   }
 
+  const { audioSourceMp3, audioSourceAac, audioSourceOgg } = meta
+  const audioSource = audioSourceMp3 || audioSourceAac || audioSourceOgg ? {
+    mp3: audioSourceMp3,
+    aac: audioSourceAac,
+    ogg: audioSourceOgg
+  } : null
+
   return {
     series,
     dossier: resolver(meta.dossier),
     format: resolver(meta.format),
-    discussion: resolver(meta.discussion)
+    discussion: resolver(meta.discussion),
+    audioSource
   }
 }
 
```

Comment:
metaFieldResolver is only done on publish but audioSource should be available before. therefor and  because what you do here is a simple sync transform, I recommend you put it in the main Meta resolver in: servers/publikator/graphql/resolvers/Meta.js.

Reply:
Sorry the right place is `getMeta` in packages/documents/lib/meta.js
This works everywhere (publikator- and republik-api).